### DISCUSSION TITLE
Follow mouse, but do not move the focus to a neighboring application outside the visible screen boundary

### DISCUSSION THREAD

**Discussion by IVKLD** [YaLTeR/niri#484] [Ideas]
Add an option whereby using "mouse tracking" the focus of a window would switch when hovering over it, BUT if the application is expanded to the full width of the screen, it would not switch to another window when hovering over the border of the application. 

I like the option where I can switch focus on mouseover, but it annoys me when that focus causes me to fly to the neighboring window that is to the right. 

If it's not clear to me, I can attach a video.

**Comment by YaLTeR** [comment]
Hmm, I see what you mean, but maybe it should be more of, if the target window is visible by less than X% then don't switch

**Comment by salman-farooq-sh** [comment]
It's also happening with scale set to 1.9 on a 3840*2400 display with border and focus ring off with gaps set to 0. Window on the right of the full width window gets focus when mouse touches the right edge of the display. Doesn't happen with scale 2.0 or window on left.

**Comment by YaLTeR** [comment]
Let's discuss in https://github.com/YaLTeR/niri/discussions/491
